operator/pkg/apis/controlplane/v1alpha1/controlplane.go

```diff
@@ -56,6 +56,7 @@ type Etcd struct {
 // custom flags for components running on master nodes like apiserver, KCM and
 // scheduler.
 type MasterSpec struct {
+	KeyId             *string    `json:"keyId,omitempty"`
```

Comment:
Sorry to keep picking on this, but given the context, I think we probably need to specify KMSKeyId. It's currently ambiguous e.g. SSHKeyId